tasks/src/sentence-similarity/data.ts

```diff
@@ -66,25 +66,25 @@ const taskData: TaskData = {
 	libraries: TASKS_MODEL_LIBRARIES["sentence-similarity"],
 	metrics:   [
 		{
-			description: "The reciprocal rank is a measure used to rank the relevancy of documents given a set of documents. Reciprocal Rank is the reciprocal of the rank of the document retrieved, meaning, if the rank is 3, the Reciprocal Rank is 0.33. If the rank is 1, the Reciprocal Rank is 1.",
```

Comment:
The metric is called "reciprocal rank" so I wouldn't change it. Relevancy is the right term grammatically.